network/policy/policy_windows.go

```diff
@@ -117,6 +116,45 @@ func IsPolicyTypeOutBoundNAT(policy Policy) bool {
 	return false
 }
 
+// IsPolicyTypeNAT returns true if the policy type is NAT
+func IsPolicyTypeNAT(policy Policy) bool {
+	if policy.Type == EndpointPolicy {
+		var endpointPolicy hcn.EndpointPolicy
+		if err := json.Unmarshal(policy.Data, &endpointPolicy); err != nil {
+			return false
+		}
+		if endpointPolicy.Type == "NAT" {
```

Comment:
I think line 126 is an issue - you changed the Type from NAT to hnsv2.PortMapping in func getPoliciesFromRuntimeCfg. 
Here you need to check if type is hnsv2.PortMapping and not NAT

Reply:
Correct! I seem to have missed this :/